modules/remotecontrol/Controller.js

```diff
@@ -123,14 +140,13 @@ class Controller {
      * @param {Object} event the remote control event.
      */
     _sendRemoteControlEvent(event) {
+        if(!this.enabled)
```

Comment:
Is it expected that _sendRemoteControlEvent will be called if Controller is not enabled ? I would rather see here some error logged or thrown in such case instead of ignoring it silently.